eventstore/whatsapp_actions.py

```diff
@@ -78,6 +79,30 @@ def handle_event(event):
     if event.is_hsm_error:
         handle_whatsapp_hsm_error(event)
 
+    if event.fallback_channel is True:
+        handle_fallback_delivery_error(event)
+
+
+def handle_fallback_delivery_error(event):
```

Comment:
I would change this to `handle_fallback_event`, since it's not only error events going here, but also success events.